contracts/storage/FeeCollectorStorage.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity =0.8.9;
+
+contract FeeCollectorStorageV1 {
+    // Any variables that would implicitly implement an IMarginEngine function if public, must instead
+    // be internal due to limitations in the solidity compiler (as of 0.8.12)
+    mapping(address => uint256) internal defaultFund;
+    mapping(address => uint256) internal protocolFees;
+    bool internal defaultFundPaused;
+}
+
+contract FeeCollectorStorage is FeeCollectorStorageV1 {
+    // Reserve some storage for use in future versions, without creating conflicts
+    // with other inheritted contracts
+    // todo: figure out the slots amount
+    uint256[69] private __gap; // total storage = 100 slots, including structs
```

Comment:
actual size and comment are not matching

Reply:
I gave it 50 slots, buffer of 47 [cbe77ae](https://github.com/Voltz-Protocol/voltz-core/pull/266/commits/cbe77ae66a92b88405421de1c72b1ee0227276a1)